lib/Listener/BeforeTemplateRenderedListener.php

```diff
@@ -102,5 +107,9 @@ public function handle(Event $event): void {
 		\OCP\Util::addScript("nmctheme", "nmctheme-conflictdialog");
 		\OCP\Util::addScript("nmctheme", "nmctheme-tooltip");
 		\OCP\Util::addScript("nmctheme", "nmctheme-shareicons");
+
+		$csp = new \OCP\AppFramework\Http\ContentSecurityPolicy();
+		$csp->addAllowedFrameDomain('\'self\'');
+		$response->setContentSecurityPolicy($csp);	
```

Comment:
This time, an additional CSP rule for frames is set. I could imagine that this is the only rule that actually has effect?
But why for frames?